server/utils/applications/applicationIdentityBar.test.ts

```diff
@@ -1,14 +1,24 @@
 import { applicationFactory, personFactory, userDetailsFactory } from '../../testutils/factories'
 import { applicationIdentityBar, applicationMenuItems, applicationTitle } from './applicationIdentityBar'
 import paths from '../../paths/apply'
+import config from '../../config'
 import { displayName } from '../personUtils'
 
 describe('applicationIdentityBar', () => {
   describe('applicationTitle', () => {
+    afterEach(() => {
+      config.flags.oneApplication = false
```

Comment:
Same as above: if the main config specifies a default of `true` for this flag, it would be overridden by this. Safer to first grab the actual value first, then use it to reset things.